packages/server/modules/serverinvites/domain/operations.ts

```diff
@@ -0,0 +1,114 @@
+import { ServerRoles, StreamRoles } from '@speckle/shared'
```

Comment:
I think there's 2 issues mixed up here: 1) Calling types helpers 2) Some types for some reason being kept separate from other types.

We can treat types as a new category, if you don't want to call them helpers - keep them in a `types` folder or something, that should address 1) (altho I don't mind them being treated as helpers).

The 2nd issue still stands, tho - what is the real practical purpose for having these types be separate from all of the other ones in their own `domain` folder? Note that they can still be in their own file, if needed, just together with all of the other type modules in `helpers` or `types`

Reply:
I think we could double-down a bit further here, even, and group `/domain` by entity. Something like `domain/serverInvites.ts` could have the definitions for:

- What a server invite is
- What the module allows you to do with it (services?)

Instead of pivoting on types/operations (where I'm sure some modules might have a massive `operations.ts` file)